src/lib/utils/sqlite_wrapper.hpp

```diff
@@ -31,33 +30,50 @@ class SQLiteWrapper final {
    * @param table      The table to load into sqlite
    * @param tablename  The desired table name
    */
-  void create_table(const Table& table, const std::string& table_name);
+  void create_sqlite_table(const Table& table, const std::string& table_name);
 
-  /*
-   * Executes a sql query in the sqlite database context and returns a Hyrise table.
-   *
-   * @param sql_query Query to be executed
-   * @returns An opossum Table containing the results of the executed query
-   */
-  std::shared_ptr<Table> execute_query(const std::string& sql);
+  class Connection final {
+   public:
+    explicit Connection(const std::string& uri);
+
+    Connection(const Connection&) = delete;
+    Connection(Connection&&) noexcept;
+    Connection& operator=(const Connection&) = delete;
+    Connection& operator=(Connection&&) = delete;
+    ~Connection();
+
+    sqlite3* db{nullptr};
+
+    /*
+     * Executes a sql query in the sqlite database context and returns a Hyrise table.
+     *
```

Comment:
The function description below reveals that this function invokes Hyrise parts. You should explain this here in short – why does this function invoke Hyrise parts?

Reply:
Now: "Execute an SQL statement on the wrapped sqlite db and ignores the result (i.e., does not convert it to a Hyrise table)"